cultural-fund-map/main.js

```diff
@@ -1,8 +1,51 @@
-var map = L.map('map').setView([39.967, -75.237], 13);
+var map = L.map('map').setView([39.949669, -75.164361], 13);
 
 L.tileLayer('https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token={accessToken}', {
     attribution: 'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, <a href="http://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery © <a href="http://mapbox.com">Mapbox</a>',
     maxZoom: 18,
     id: 'cahdeemer.n2p8d0mj',
     accessToken: 'pk.eyJ1IjoiY2FoZGVlbWVyIiwiYSI6ImIwYzM3OGYyZTBmYmZlMWJmMzQ5OTNmZWRjMTA3NjNmIn0.4lMKKySybu846ym7-BNbYA'
-}).addTo(map);
\ No newline at end of file
+}).addTo(map);
+
+var dataSuccess = function( jsonData ) {
+	console.log(jsonData);
+	var layerOptions = {
+		pointToLayer: function(featureData, latlng) {
+			var markerOptions = {
+				color: '#f1f2f2',
+				fillOpacity: 0.9,
+				fillColor: (function(x) {
+					if (x > 49999)
+					return '#002C35';
+					if (x > 24999)
+					return '#006e85';
+					if (x > 14999)
+					return '#4d9aaa'
+					else
+					return '#99c5c3';
+				}
+				(featureData.properties.grantAmount)),
+				radius: (function(x) {
+					if (x > 1500000)
+					return 15;
+					if (x > 400000)
+					return 12;
+					if (x >150000)
+					return 9;
+					if (x > 50000)
+					return 6
+					else 
+					return 3;
+				}
+				(featureData.properties.orgBudget))
```

Comment:
I know this self executing function was my idea at the rush of the end of class, but I think it makes much more readable code to extract it to a named function and call it traditionally; it's easier to see what value you're actually passing in.  A version might look like this (though I didn't port the improvements to the function from my previous comment):

```javascript
        var getColor = function(x) {
                if (x > 49999)
                return '#002C35';
                if (x > 24999)
                return '#006e85';
                if (x > 14999)
                return '#4d9aaa'
                else
                return '#99c5c3';
            },

            getRadius = function(x) {
                if (x > 1500000)
                return 15;
                if (x > 400000)
                return 12;
                if (x >150000)
                return 9;
                if (x > 50000)
                return 6
                else 
                return 3;
            },

			markerOptions = {
				color: '#f1f2f2',
				fillOpacity: 0.9,
				fillColor: getColor(featureData.properties.grantAmount),
				radius: getRadius(featureData.properties.orgBudget)
			}

			return L.circleMarker(latlng, markerOptions);
		}
```

Again, this is functionally equivalent to the existing code, but I find that it reads better over time, which is what I consider to be the most important characteristic of code - the idea is that you write it once but read it often, so better to optimize for that when you can.  

It also helps reinforce that `markerOptions` is a simple object, which is a little obscured with the function definitions mixed up in with the property definitions of the object.